Sources/FluentMySQL/DefaultValue.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+
+public enum DefaultValue {
```

Comment:
There should at least be a `raw` case here to pass a custom function. Event better, have `DefaultValue` be a struct and have all of the cases be static conveniences :)

Also, should be `MySQLDefaultValue` for namespacing

```swift
struct MySQLDefaultValue {
    public var value: String

    public static var null: MySQLDefaultValue { return .init(value: "NULL") }
    ... etc
}
```